tests/Curl/CurlTest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brick\Std\Tests\Curl;
+
+use Brick\Std\Curl\Curl;
+use PHPUnit\Framework\TestCase;
+
+class CurlTest extends TestCase
+{
+    public function testConstructorShouldInitialCurl()
+    {
+        $curl = new Curl();
+
+        $this->assertInternalType('object', $curl);
+    }
+
+    public function testCurlClassCanBeCloned()
+    {
+        $curl = new Curl();
+        $newCurl = clone $curl;
+
+        $this->assertInstanceOf(Curl::class, $newCurl);
```

Comment:
Same, `clone` will always return an object, of the same class. There's no need to keep this test.

Reply:
I will remove this test.